types/node/index.d.ts

```diff
@@ -1912,56 +1905,56 @@ declare module "https" {
     import * as http from "http";
     import { URL } from "url";
 
-    export type ServerOptions = tls.SecureContextOptions & tls.TlsOptions;
+    type ServerOptions = tls.SecureContextOptions & tls.TlsOptions;
 
-    export type RequestOptions = http.RequestOptions & tls.SecureContextOptions & {
+    type RequestOptions = http.RequestOptions & tls.SecureContextOptions & {
         rejectUnauthorized?: boolean; // Defaults to true
         servername?: string; // SNI TLS Extension
     };
 
-    export interface AgentOptions extends http.AgentOptions, tls.ConnectionOptions {
+    interface AgentOptions extends http.AgentOptions, tls.ConnectionOptions {
         rejectUnauthorized?: boolean;
         maxCachedSessions?: number;
     }
 
-    export class Agent extends http.Agent {
+    class Agent extends http.Agent {
         constructor(options?: AgentOptions);
         options: AgentOptions;
     }
 
-    export class Server extends tls.Server {
+    class Server extends tls.Server {
         setTimeout(callback: () => void): this;
         setTimeout(msecs?: number, callback?: () => void): this;
         timeout: number;
         keepAliveTimeout: number;
     }
 
-    export function createServer(options: ServerOptions, requestListener?: (req: http.IncomingMessage, res: http.ServerResponse) => void): Server;
-    export function request(options: RequestOptions | string | URL, callback?: (res: http.IncomingMessage) => void): http.ClientRequest;
-    export function request(url: string | URL, options: RequestOptions, callback?: (res: http.IncomingMessage) => void): http.ClientRequest;
-    export function get(options: RequestOptions | string | URL, callback?: (res: http.IncomingMessage) => void): http.ClientRequest;
-    export function get(url: string | URL, options: RequestOptions, callback?: (res: http.IncomingMessage) => void): http.ClientRequest;
-    export var globalAgent: Agent;
+    function createServer(options: ServerOptions, requestListener?: (req: http.IncomingMessage, res: http.ServerResponse) => void): Server;
+    function request(options: RequestOptions | string | URL, callback?: (res: http.IncomingMessage) => void): http.ClientRequest;
+    function request(url: string | URL, options: RequestOptions, callback?: (res: http.IncomingMessage) => void): http.ClientRequest;
+    function get(options: RequestOptions | string | URL, callback?: (res: http.IncomingMessage) => void): http.ClientRequest;
+    function get(url: string | URL, options: RequestOptions, callback?: (res: http.IncomingMessage) => void): http.ClientRequest;
+    const globalAgent: Agent;
```

Comment:
I think this is not const. It may be a valid usecase to modify the global agent in an application.